### ISSUE TITLE
7.11.2 TopBar button color issue on IOS 14

### ISSUE THREAD

**Issue by alenmu** [wix/react-native-navigation#7041]
## 🐛 Bug Report

When upgrading from 7.6.0 to 7.11.2, it appears that setting color for left/right buttons in the TopBar is set to blue by default. When using a png icon the whole icon gets painted to the set color. 
Using `disableIconTint: true` does not solve the issue and as far i can see it was removed from the official docs but it can still be found in the types. This issue does not appear on android devices.

### Have you read the [Contributing Guidelines on issues](https://wix.github.io/react-native-navigation/docs/meta-contributing/)?

Yes

## To Reproduce

(Write your steps here:)

1. Load a nav button icon to use (a png icon)
2. Construct a component in the following manner:
    topBar: {
      rightButtons: [
        {
          id: premiumScreenTestIDs.close,
          testID: premiumScreenTestIDs.close,
          icon: iconsList.topBarCloseFill
        }
      ],
      backButton: { visible: false }
    }

3. Observe that the buttons are 'navigation' blue on iOS simulator/real devices. If you set an icon color it will paint over the whole icon in that color.

## Expected behavior

Expected the buttons to keep the original png properties as is on Android emulators/devices

## Actual Behavior

The whole png button gets painted over.

## Your Environment

- React Native Navigation version: 7.11.2
- React Native version: 0.64.0
- Platform(s) (iOS, Android, or both?): iOS
- Device info (Simulator/Device? OS version? Debug/Release?): Simulator/Real device running IOS 14

## Reproducible Demo

Can create one if necessary, but seems fairly easy to reproduce with any existing project.
![110674184-f005ec00-81d1-11eb-9960-ac9c64dbfc5a](https://user-images.githubusercontent.com/17156991/111599822-0105bd00-87d1-11eb-8bba-6c1a01162709.png)

**Comment by elsurudo** [comment]
@alenmu have you found a workaround by any chance?

**Comment by elsurudo** [comment]
FYI on iOS I'm pretty sure these nav bar buttons can only have one tint color, so what I did was this:

```
MyView.options = {
  topBar: {
    title: {
      color: 'white'
    },
    background: {
      color: 'black'
    },
    leftButtons: [
      {
        id: 'reportButton',
        text: "Report",
        icon: require('../../img/icon/toolbar-report.png'),
        color: 'white'
      }
    ],
    rightButtons: [
      {
        id: 'settingsButton',
        text: "Settings",
        icon: require('../../img/icon/toolbar-settings.png'),
        color: 'white'
      }
    ]
  }
}
```

to set the color explicitly.

**Comment by alenmu** [comment]
> @alenmu have you found a workaround by any chance?

Nope, still nothing

**Comment by alenmu** [comment]
> FYI on iOS I'm pretty sure these nav bar buttons can only have one tint color, so what I did was this:
> 
> ```
> MyView.options = {
>   topBar: {
>     title: {
>       color: 'white'
>     },
>     background: {
>       color: 'black'
>     },
>     leftButtons: [
>       {
>         id: 'reportButton',
>         text: "Report",
>         icon: require('../../img/icon/toolbar-report.png'),
>         color: 'white'
>       }
>     ],
>     rightButtons: [
>       {
>         id: 'settingsButton',
>         text: "Settings",
>         icon: require('../../img/icon/toolbar-settings.png'),
>         color: 'white'
>       }
>     ]
>   }
> }
> ```
> 
> to set the color explicitly.

Yep, but that doesn't solve the issue. You still cant import a png that has 2 different tint colors on it.

**Comment by tastyqbit** [comment]
@yogevbd Hi! I found you as a dev in one of the previous tickets. Would it be possible to have a look at this? Sorry for the tag, I am not sure how to get visibility on a bug that hasn't been triaged yet!

**Comment by madandrija** [comment]
@yogevbd - I'm trying to find where this is being set, but can't find my way around the code; if there's no time to fix this properly (or if this is intended behaviour), could you please let us know/give us a direction? Thanks

**Comment by yedidyak** [comment]
Closing this as part of a maintainer cleanup of open issues older than one year.

Decision: Closed: stale/non-critical; no evidence it is a critical current-version RNN issue.

If this still reproduces on the latest React Native Navigation with a minimal reproduction, please open a new issue with current RN/RNN versions and repro steps.
